index.js

```diff
@@ -45,11 +45,25 @@ function setupMethods (soljson){
 		return JSON.parse(result);
 	}
 
+        var link = function(bytecode, libraries) {
+                for (var libraryName in libraries) {
+                        var libLabel = '__' + libraryName + Array(39 - libraryName.length).join('_');
```

Comment:
Actually what should happen in that case? What is happening within Solidity? Error or truncate?